Rymera-Web-Co/wcvendors
# Added: Migrate vendor shop

### All Submissions:






### Changes proposed in this Pull Request:


Mirgrated functions:
- [x] Vendor Store
	- [x] Store query 
	- [x] sold by order meta 
	- [x] page header title update
- [x] Single product page 
	- [x] Store info tab 
	- [x] Sold by label 
- [x] Cart 
	- [x] Sold by label 

### How to test the changes in this Pull Request:

1. Can view vendor shop page
2. See shop header, single product shop header, store info tab in single product page
3. Sold by label in cart, order, shop loop, single product

### Other information:
